ui/pages/_document.js

```diff
@@ -37,9 +37,10 @@ class MesheryDocument extends Document {
       <Html lang="en" dir="ltr">
         <Head>
           <meta charSet="utf-8" />
+          <meta name="referrer" content="no-referrer" />
```

Comment:
```suggestion
          {/**
           * content="no-referrer" included to avoid 403 errors on Google avatars
          */}
          <meta name="referrer" content="no-referrer" />
```